src/code/templates/genome-challenge.js

```diff
@@ -30,10 +31,16 @@ export default class GenomeChallengeTemplate extends Component {
       onDrakeSubmission(targetDrake.phenotype.characteristics, userDrake.phenotype.characteristics, correct);
     };
 
-    const userDrakeStyle = userDrakeHidden ? "hiddenDrake" : "";
+    const instructionsBanner = instructions
+                                ? <div className="instructions-banner">
+                                    <div className="instructions-text">{instructions}</div>
+                                  </div>
+                                : null,
+          userDrakeStyle = !showUserDrake && userDrakeHidden ? "hiddenDrake" : "";
```

Comment:
I'm not completely clear on why we need both showUserDrake and userDrakeHidden props?